### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by ainy0315** [looi-wh/SubTranslate#1]
hi I want to know how to translate

**Comment by looi-wh** [comment]
hi. the script will first grab the current directory and scan for any subtitles (including files inside any folders. once the scanning process is completed, it will create a list of to-dos and slowly process all of them.

there will be a filter that only process files with ".english.default", but this could be turned off easily by editing the script.

edit the script to view all of the available settings. some of them are useful.

to avoid any issues, please test the script to understand it before deploying to production. 

so in short: put the script beside the files you want to translate, and run "python3 subTranslate.py"
